pkgs/servers/sql/postgresql/patches/prevent-output-cycle.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/port/Makefile b/src/port/Makefile
+index bfe1feb..1f9f0d3 100644
+--- a/src/port/Makefile
++++ b/src/port/Makefile
+@@ -152,9 +152,9 @@ pg_config_paths.h: $(top_builddir)/src/Makefile.global
+ 	echo "#define PGBINDIR \"$(bindir)\"" >$@
+ 	echo "#define PGSHAREDIR \"$(datadir)\"" >>$@
+ 	echo "#define SYSCONFDIR \"$(sysconfdir)\"" >>$@
+-	echo "#define INCLUDEDIR \"$(includedir)\"" >>$@
+-	echo "#define PKGINCLUDEDIR \"$(pkgincludedir)\"" >>$@
+-	echo "#define INCLUDEDIRSERVER \"$(includedir_server)\"" >>$@
++	echo "#define INCLUDEDIR \"/dev/null/include\"" >>$@
```

Comment:
Doesn't this break with e.g. https://github.com/postgres/postgres/blob/aa210e0c121eb8f58c86d4fcc833a5a6fbb6f5a9/src/interfaces/ecpg/preproc/ecpg.c#L256 since `get_include_path` uses `INCLUDEDIR` generated by this makefile?

Reply:
It will return the wrong value but the assumption is that nothing uses the function for anything other than printing it in some build information pages.